content/hardware/04.pro/boards/portenta-x8/features.md

```diff
@@ -9,7 +15,7 @@
   </Feature>
 
   <Feature title="STM32H747XI dual Cortex®-M7+M4 32bit low power Arm® MCU" image="mcu">
-    X8's integrated microcontroller is the dual core STM32H747 including a Cortex® M7 running at 480 MHz and a Cortex® M4 running at 240 MHz. The M4 core is available for real time control of motors and other time-critical machinery at a barebones level. The M7 core acts as a mediator between the peripherals and the i.MX 8M Mini and runs a proprietary firmware inaccessible to the User.
+    X8's integrated microcontroller is the dual-core STM32H747 including a Cortex® M7 running at 480 MHz and a Cortex® M4 running at 240 MHz. The M4 core is available for real-time control of motors and other time-critical machinery at a barebones level. The M7 core acts as a mediator between the peripherals and the i.MX 8M Mini and runs a proprietary firmware inaccessible to the User.
```

Comment:
I am just specifying a little bit more due to the comments we received from users: 

X8's integrated microcontroller is the dual-core STM32H747, including a Cortex® M7 running at 480 MHz and a Cortex® M4 running at 240 MHz. The M4 core can be programmed through the Arduino IDE to run sketches for multiple real-time applications, like control of motors or other time-critical machinery at a barebones level. The M7 core runs an Arduino custom firmware, normally invisible to the User, able to map all its peripherals as Linux devices.

Reply:
hyphens make sense and the small change related to wording too.

Marta text incorporated.